src/App.tsx

```diff
@@ -88,78 +89,82 @@ const AppRoutes = ({ base, resCookie }: AppProps) => {
     <AlertsProvider>
       <BaseNameProvider value={base}>
         <AuthenticationContext initialValue={resCookie}>
-          <SnackbarProvider>
-            <Routes>
-              <Route path="/" element={<Layout />}>
-                <Route index element={<WelcomePage />} />
-                <Route path="subjects" element={<AllSubjectsPage />} />
-                <Route path="search" element={<SearchPage />} />
-                <Route path="utdanning/:programme" element={<ProgrammePage />}>
-                  <Route path=":grade" element={null} />
-                </Route>
-                <Route path="podkast">
-                  <Route index element={<PodcastSeriesListPage />} />
-                  <Route path=":id" element={<PodcastSeriesPage />} />
-                </Route>
-                <Route
-                  path="article/:articleId"
-                  element={<PlainArticlePage />}
-                />
-                <Route
-                  path="learningpaths/:learningpathId"
-                  element={<PlainLearningpathPage />}>
-                  <Route path="steps/:stepId" element={null} />
-                </Route>
-                <Route path="subject:subjectId/topic:topicId/resource:resourceId">
-                  {resourceRoutes}
-                </Route>
-                <Route path="subject:subjectId/topic:topic1/topic:topicId/resource:resourceId">
-                  {resourceRoutes}
-                </Route>
-                <Route path="subject:subjectId/topic:topic1/topic:topic2/topic:topicId/resource:resourceId">
-                  {resourceRoutes}
-                </Route>
-                <Route path="subject:subjectId/topic:topic1/topic:topic2/topic:topic3/topic:topicId/resource:resourceId">
-                  {resourceRoutes}
-                </Route>
-                <Route path="subject:subjectId/topic:topic1/topic:topic2/topic:topic3/topic:topic4/topic:topicId/resource:resourceId">
-                  {resourceRoutes}
-                </Route>
-                <Route path="subject:subjectId" element={<SubjectRouting />}>
-                  <Route path="topic:topicId" element={null} />
-                  <Route path="topic:topic1" element={null}>
+          <UserSettingsProvider initialValue={resCookie}>
```

Comment:
Gir egentlig ikke mening å serverside rendre noe som krever bruker-settings lenger (med mindre det er en del av path'en), så tenker vi gjør lurt i å ikke sende cookien gjennom her fra server, og heller bruke `NoSSR` e.l for å få det til å fungere på server.